pkg/exporter/exporter.go

```diff
@@ -118,6 +119,7 @@ func (e *Exporter) Scrape(ch chan<- prometheus.Metric) {
 				"subject":         cert.Subject.String(),
 				"dns_names":       strings.Join(cert.DNSNames, ","),
 				"email_addresses": strings.Join(cert.EmailAddresses, ","),
+				"node":            os.Getenv("HOSTNAME"),
```

Comment:
Please make use of [`os.Hostname()`](https://golang.org/pkg/os/#Hostname) instead

Reply:
done